tests/pkg_tensorflow-lite/mnist/main_functions.cc

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ *
+ * @file
+ * @brief       TensorFlow Lite MNIST MLP inference functions
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#include <stdio.h>
+#include "tensorflow/lite/micro/micro_error_reporter.h"
+#include "tensorflow/lite/micro/micro_interpreter.h"
+#include "tensorflow/lite/micro/kernels/micro_ops.h"
+#include "tensorflow/lite/micro/micro_mutable_op_resolver.h"
+#include "tensorflow/lite/schema/schema_generated.h"
+#include "tensorflow/lite/version.h"
+
+#include "blob/digit.h"
+#include "blob/model.tflite.h"
+
+#define THRESHOLD       (0.5)
+
+// Globals, used for compatibility with Arduino-style sketches.
```

Comment:
I don't know if we have coding conventions for `cpp` files, do we enforce comment formats?

Reply:
Since the code is written in C++, I think it makes more sense to C++ comment style.